packages/grafana-runtime/src/services/live.ts

```diff
@@ -37,13 +37,18 @@ export interface GrafanaLiveSrv {
   /**
    * Configure a channel with the given setup
    */
-  initChannel<T>(channel: string, handler: ChannelHandler<T>): void;
+  initChannel<T>(channel: string, handler: ChannelHandler<T>): Observable<T>;
 
   /**
    * Subscribe to activity on a given channel
    */
   getChannelStream<T>(channel: string): Observable<T>;
 
+  /**
+   * Unsubscribe all local references to a channel
+   */
+  closeChannelStream(channel: string): void;
```

Comment:
is this needed? can't unsubscribing to the initChannel Observable do this?

Reply:
removed